dragula.js

```diff
@@ -94,7 +94,8 @@ function dragula (initialContainers, options) {
     _moveX = e.clientX;
     _moveY = e.clientY;
 
-    var ignore = whichMouseButton(e) !== 1 || e.metaKey || e.ctrlKey;
+    var whichMouse = whichMouseButton(e);
+    var ignore = (whichMouse !== 0 && whichMouse !== 1) || e.metaKey || e.ctrlKey;
```

Comment:
Could you change this check to `(button !== 0 && e.type !== 'mousedown') || button !== 1 || e.metaKey || e.ctrlKey`, and rename `whichMouse` to `button`? I think that would still fix the mobile thing, and it should also fix the input regressions it'd otherwise introduce.